test/test_utils/sagemaker.py

```diff
@@ -332,13 +334,20 @@ def execute_local_tests(image, pytest_cache_params):
                                                      executable="/bin/bash")
                     pytest_cache_util.upload_pytest_cache_from_ec2_to_s3(ec2_conn_new, path, **pytest_cache_params)
                     if 'failures="0"' not in str(output):
-                        raise ValueError(f"Sagemaker Local tests failed for {image}")
+                        if is_nightly_context():
+                            print(f"\nSuppresed Failed Nightly Sagemaker Local Tests")
+                        else:
+                            raise ValueError(f"Sagemaker Local tests failed for {image}")
             else:
-                ec2_conn.run(pytest_command)
-                print(f"Downloading Test reports for image: {image}")
-                ec2_conn.get(ec2_test_report_path, os.path.join("test", f"{job_type}_{tag}_sm_local.xml"))
-    except:
-        print(f"Exception {sys.exc_info()[0]} occurred")
+                try:
+                    ec2_conn.run(pytest_command)
+                    print(f"Downloading Test reports for image: {image}")
```

Comment:
As discussed on call, please go ahead and change this to a `warn=True` and use the output Result object to figure out further steps.